### DISCUSSION TITLE
Setting the progress bar to the value 0 doesn't really make it look like 0

### DISCUSSION THREAD

**Discussion by duruburak** [TomSchimansky/CustomTkinter#660] [Q&A]
How can I make the progress bar look like there is no progress in it?

```
screen = customtkinter.CTk()
screen.geometry("1100x900")
frame_1 = customtkinter.CTkFrame(master=screen)
frame_1.pack(pady=20, padx=60, fill="both", expand=True)
progress_label =  customtkinter.CTkProgressBar(frame_1, fg_color="white", width=500, height=40, corner_radius=100)
progress_label.pack(pady=400, padx=5)
progress_label.set(0)
screen.mainloop()
```

![progressBar](https://user-images.githubusercontent.com/100048974/204093163-8c04eb4e-6e2c-4e32-876e-a817d0732c02.png)

**Comment by Akascape** [comment]
@duruburak Make the `progress_color="white"` (where white is the fg_color of the Progress Bar).

Then change that to normal progress color whenever it is used.
`progress_label.configure(progress_color="green")`

**Reply by duruburak** [reply]
I actually thought of it but I wondered if there was any other way.
